components/roster/heroes/Heroes.tsx

```diff
@@ -0,0 +1,283 @@
+                    {isAscensionLevelValid(hero.ascension) && (
+                      <div className="z-10 absolute -bottom-4 w-[86%] h-full bg-slate-700 rounded-b-md mx-auto left-0 right-0 px-1">
+                        <div className="flex flex-row h-full items-end justify-center pb-1">
+                          {Array.from(
+                            { length: hero.exEquipment / 5 + 1 },
+                            (_, i) => (
+                              <div key={i} className="flex flex-row">
+                                <Image
+                                  src={rosterImages.equipmentStar}
+                                  alt="Equipment Star"
+                                  width={13}
+                                  height={13}
+                                />
+                              </div>
+                            ),
+                          )}
+                        </div>
+                      </div>
+                    )}
+                  </CardContent>
+                </Card>
+              </DropdownMenuTrigger>
+              <DropdownMenuContent className="w-56">
+                <DropdownMenuLabel>{hero.name}</DropdownMenuLabel>
+                <DropdownMenuSeparator />
+                <div className="px-2 py-1.5">
+                  <div className="flex items-center space-x-2">
+                    <Checkbox
+                      id="unlock-checkbox"
+                      checked={hero.unlocked}
+                      onCheckedChange={(checked) =>
+                        dispatch({
+                          property: "unlock",
+                          value: checked.toString(),
+                          hero: hero,
+                        })
+                      }
+                    />
+                    <Label
+                      htmlFor="unlock-checkbox"
+                      className="text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
+                    >
+                      {hero.unlocked ? (
+                        <span className="flex items-center">
+                          <Unlock className="h-4 w-4" />
+                          Unlocked
+                        </span>
+                      ) : (
+                        <span className="flex items-center">
+                          <Lock className="h-4 w-4" />
+                          Locked
+                        </span>
+                      )}
+                    </Label>
+                  </div>
+                </div>
+                <DropdownMenuSeparator />
+                {hero.unlocked ? (
+                  <>
+                    <CustomSelectSubmenu
+                      hero={hero}
+                      heroProperty="ascension"
+                      options={
+                        hero.tier === "S"
+                          ? Object.values(AscensionLevel).splice(2)
+                          : Object.values(AscensionLevel)
```

Comment:
why are we splicing this when a hero's tier is S?

Reply:
S Tier can't go below Epic, so it cuts off Elite and Elite+